package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-basic-dropdown",
-  "version": "0.12.0-beta.22",
```

Comment:
can you revert this line? It is incremented automatically what I run `ember release`